website/blog/2023-11-23-coding-llm-leaderboard/index.mdx

```diff
@@ -0,0 +1,36 @@
+---
+authors: [ meng ]
+
+tags: [tech design]
+
+image: ./leaderboard.png
+
+---
+# Introducing the Coding LLM Leaderboard
+
+In our previous post on [Cracking the Coding Evaluation](../13/model-evaluation), we shed light on the limitations of relying on HumanEval pass@1 as a code completion benchmark.
+In response, we've launched the [Coding LLMs Leaderboard](http://leaderboard.tabbyml.com), embracing "Next Line Accuracy" as a metric inspired by academic works such as [RepoCoder](https://arxiv.org/abs/2303.12570), [RepoBench](https://arxiv.org/abs/2306.03091), and [CCEval](https://arxiv.org/abs/2310.11248).
+
+![Leaderboard](./leaderboard.png)
+
+But what exactly is line accuracy? In code completion, it predicts a block of code spanning multiple lines. While achieving full block match accuracy may seem ideal, it's often considered "too sparse" as a revealing metric.
+Next line accuracy, on the other hand, serves as a dependable proxy for overall block match accuracy.
+
+![Next Line Accuracy](./next-line-accuracy.png)
+
+<p style={{"text-align": "center"}}>
+
+*Only content inside red box are used to compared with ground truth to compute accuracy metric*
+
+</p>
+
```

Comment:
Maybe add a short paragraph talking about why "next-line accuracy" is a better metric to monitor?
For example, why it's more consistent with users' real experience in code completion?